### ISSUE TITLE
Installing multiple ports in one command

### ISSUE THREAD

**Issue by j-minster** [amake/macports.el#3]
Hello! Thanks for your nice emacs package. I would like to know if this feature is present or if it needs hacking: 

Being able to install multiple packages in one command. 

Currently I am doing:

1.  `M-x macports`
2. `i` (for `install`)
3. Selecting a port using completing-read (vertico)

I try and hit 'comma' for completing-read-multiple, but no dice. 
Is it something there is plumbing for, which I just need to tell vertico about? 

Kind regards, 

Jack.

**Comment by amake** [comment]
No, I don't expect multiple selection to work as-is, since I am calling `completing-read`, not `completing-read-multiple`.

I didn't even know `completing-read-multiple` existed. I can switch to it easily enough, but I find the UX to be worse since it doesn't immediately begin filtering the list—you have to know you can prompt for completion.

Is there a way to make the UX more like `completing-read`?

**Comment by amake** [comment]
Also is there an idiom for the prompt string that would let the user know that multiple selection is allowed?

**Comment by j-minster** [comment]
I found this in the [vertico](https://github.com/minad/vertico) README, which adds a CRM indicator in the minibuffer: 

```emacs-lisp
(defun crm-indicator (args)
    (cons (format "[CRM%s] %s"
                  (replace-regexp-in-string
                   "\\`\\[.*?]\\*\\|\\[.*?]\\*\\'" ""
                   crm-separator)
                  (car args))
          (cdr args)))
  (advice-add #'completing-read-multiple :filter-args #'crm-indicator)
```

I also found that `denote` uses CRM in some places ([here](https://github.com/protesilaos/denote/blob/c23b37c23a6f2b99ae021e052c07894ae82b97dc/denote.el#L891), for example). I've attached a screen-recording of this functionality. All I'm doing in the screen-recording is typing the search prefix, picking the item with TAB, then hitting 'comma' to move onto the next selection. The list is 'pre-populated' without an initial prompt from me. 

If you're too busy I can hack this together, I need some projects to improve my elisp anyhow. :)


https://user-images.githubusercontent.com/71196960/234838435-81d6ebf5-6bb5-43b7-939b-6e74367e06d6.mov
